Source/engine/render/clx_render.cpp

```diff
@@ -647,6 +647,58 @@ void ClxApplyTrans(ClxSpriteSheet sheet, const uint8_t *trn)
 	}
 }
 
+bool IsPointWithinClx(Point position, ClxSprite clx)
+{
+	const uint8_t *src = clx.pixelData();
+	const uint8_t *end = src + clx.pixelDataSize();
+	const uint16_t width = clx.width();
+
+	int xCur = 0;
+	int yCur = clx.height() - 1;
+	while (src < end) {
+
+		if (yCur != position.y) {

```

Comment:
This can be an infinite loop if there are consecutive transparent lines because `skipSize.wholeLines` in that case can be more than 1 and skip through the target line.

Reply:
Oh sorry that's an `if` not a while.

You can do `if (yCur > position.y) return false;` to break out early if any lines were skipped.

Reply:
fixed 🙂 
Thanks for looking at the sprite-code. I'm not 100% sure I got it right. 😅